_posts/2015-01-03-submissions.md

```diff
@@ -7,25 +7,25 @@ fa-icon: file-text-o
 icon-title: true
 ---
 
-We invite presenters to submit high-quality work with sufficient background
+We invite presenters to submit talk proposals to present high-quality work with sufficient background
 material to be clear to the HPC, Big Data, and/or Data Science communities.
 Talks proposals should be submitted through the [FOSDEM Pentabarf
-server](https://penta.fosdem.org/submission/FOSDEM16). Submissions should include:
+server](https://penta.fosdem.org/submission/FOSDEM16). Submissions must include:
 
 - Abstract
 - Session type
 - Session length
-- Participants prior knowledge required / intended audience
+- Expected prior knowledge / intended audience
 - Speaker bio
 - Links to code / slides / material for the talk (optional)
 - Links to previous talks by the speaker
 
+Our intention is to have a full day of talks of about 20 minutes each, with an additional 5 minutes for questions by attendees.
+
 We would also like to note: 
 
 - Talks will be streamed live and will be recorded. By submitting a session, speakers agree to being recorded and having their talk made available.
-- All accepted talks will be about free and open source software. We highly discourage "marketing" talks.
-
-Furthermore, authors must identify any conflict of interest with the Devroom organizers.
```

Comment:
I dropped this intentionally. As organisers, we should be honest enough to make it very clear that we have a conflict of interest with certain submissions, and to review submissions objectively. FOSDEM is not an academic conference, so let's not try and scare people from submitting a talk proposal with a statement like this.